test/functional/fixtures/runner/upload/regression.test.js

```diff
@@ -0,0 +1,36 @@
+'@fixture upload regression';
+'@page ./regression.html';
+
+
+var files = ['./files/img1.jpg', './files/img2.jpg'];
+
+
+'@test'['Upload for added element'] = {
+    '1.Add element': function () {
+        act.click('#addBtn');
+    },
+
+    '2.Check upload': function () {
+        act.upload('#addedInput', files);
+    }
+};
+
+'@test'['Upload for replaced element'] = {
+    '1.Replace element': function () {
+        act.click('#replaceBtn');
+    },
+
+    '2.Check upload': function () {
+        act.upload('#addedInput', files);
+    }
+};
+
+'@test'['Upload for removed element - should fail'] = {
+    '1.Delete element': function () {
+        act.click('#deleteBtn');
+    },
+
+    '2.Check upload': function () {
+        act.upload('#newInput', files);

```

Comment:
What does this test exactly check? I see just trying to upload to non-existent element (`#newInput`). Why we need the first step then?